src/appkit.cs

```diff
@@ -15637,8 +15621,10 @@ partial interface NSView : NSDraggingDestination, NSAnimatablePropertyContainer,
 #else
 		[Internal]
 		nint _AddToolTip (CGRect aRect, NSObject anObject, IntPtr data);
-#endif
 
+		[Wrap ("this._AddToolTip (rect, (NSObject)owner, IntPtr.Zero)"]
+		nint AddToolTip (CGRect rect, INSToolTipOwner owner)
```

Comment:
It seems to me that this could be:

```csharp
[Export ("addToolTipRect:owner:userData:")]
#if !XAMCORE_4_0
nint AddToolTip (CGRect rect, NSObject owner, IntPtr userData);
#else
nint AddToolTip (CGRect rect, INSToolTipOwnwer owner, IntPtr userData);
#endif

[Wrap ("AddToolTip (rect, owner, IntPtr.Zero)")]
nint AddToolTip (CGRect rect, INSToolTipOwner owner);
```

Not really related to this PR though.

Reply:
Yeah, I only added this as I was deleting the code behind version (which had this overload).

Good point thought, I think that's better.